scanner/VULNERABILITY_VERSION

```diff
@@ -1 +1 @@
-v2
+dev
```

Comment:
This is incorrect. You can control which vuln to pull using the scanner configuration.

Reply:
But the v2 bundle doesn't contain EPSS data until we release 4.7 so I am not using v2 here. What do you think?

Reply:
And that's expected, right? This specifies the latest support schema version; Enrichment data does not require a version schema bump (as far as I remember), so we should keep this 'v2'.